src/styles/scss/abstracts/_mixins.scss

```diff
@@ -0,0 +1,61 @@
+@mixin header-shadow {
+  box-shadow: 0px 2px 10px rgba(22, 22, 22, 0.1);
+}
+
+@mixin common-shadow {
+  box-shadow: 0px 0px 10px rgba(22, 22, 22, 0.1);
+}
+
+@mixin extra-small-text-regular {
+  font-family: $type-family;
+  font-weight: $type-weight;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 18px;
+  color: $text;
+}
+
+@mixin small-text-regular {
+  font-family: $type-family;
+  font-weight: $type-weight;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: $text;
+}
+
+@mixin small-text-semi-bold {
+  font-family: $type-family;
+  font-weight: $type-weight--semi-bold;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: $text;
+}
+
+@mixin small-text-italic-regular {
+  font-family: $type-family;
+  font-weight: $type-weight--semi-bold;
+  font-style: italic;
+  font-size: 16px;
+  line-height: 24px;
+  color: $text;
+}
+
+@mixin heading-h1 {
+  font-family: $type-family;
+  font-weight: $type-weight--semi-bold;
+  font-style: normal;
+  font-size: 28px;
+  line-height: 38px;
+  color: $text;
+}
+
+@mixin heading-h2 {
+  font-family: $type-family;
+  font-weight: $type-weight--semi-bold;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: $text;
```

Comment:
Looks like all props are the same, just different values. Probably you can create general mixin here and use it 

Something like
```
@mixin custom-text (...params) {
  font-family:...
  font-weight: ...
  font-style: ...
...
}
```